spec/models/task_spec.rb

```diff
@@ -1,5 +1,43 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
+    it 'is valid with all attributes' do
+      user = FactoryBot.create(:user)
+      task = user.tasks.new(title: "aaa", content: "aaa", status: :todo)
```

Comment:
`users.tasks.new`の代わりに`FactoryBot.build`を使ってみましょう。
また、TaskのFactoryにuserオブジェクトを指定する形で作成してみてください！
```suggestion
      user = FactoryBot.create(:user)
      task = FactoryBot.build(:task, user: user)
```

Reply:
177273b